graylog2-web-interface/src/components/common/TimeUnitInput.jsx

```diff
@@ -79,8 +79,10 @@ export const extractDurationAndUnit = (duration, timeUnits) => {
  * value.
  */
 const TimeUnitInput = createReactClass({
+  // eslint-disable-next-line
```

Comment:
It looks like you are using `// eslint-disable-next-line` a few times without specifying a rule. Let's always specify the rule to ensure we.are not disabling every rule for this line and also to make it more obvious if we may can fix the rule easily.